test/unit/feirari/RewardsDistributorAdmin.test.ts

```diff
@@ -13,8 +13,13 @@ const {
     constants: { ZERO_ADDRESS }
 } = testHelpers;
 
+/// TODO
```

Comment:
Whats going on with these comments?

Reply:
Removed. It was for functions in RDA that I needed to test.